src/abstractenvironment.jl

```diff
@@ -608,6 +616,8 @@ function test_env(etype::Type{<:AbstractEnvironment}, args...; kwargs...)
             # execute a random control _trajectory and check for repeatability
             let e1 = makeenv(), e2 = makeenv()
                 actions = rand(actionspace(e1), 1000)
+                reset!(e1)
```

Comment:
env after construction and after reset! should be exactly the same (if an env overrides reset!, they need to call it in their inner constructor)